README.md

```diff
@@ -372,6 +372,7 @@ Where to discover new tools and discuss about existing ones.
 * [ML Ops: Operationalizing Data Science](https://www.oreilly.com/library/view/ml-ops-operationalizing/9781492074663/) (O'Reilly)
 * [MLOps Engineering at Scale](https://www.manning.com/books/mlops-engineering-at-scale) (Manning)
 * [Practical MLOps](https://www.oreilly.com/library/view/practical-mlops/9781098103002/) (O'Reilly)
+* [Designing Machine Learning Systems](https://www.oreilly.com/library/view/designing-machine-learning/9781098107956/) (O'Reilly)
```

Comment:
Duplicated, can you remove please?

Reply:
I'm sorry, didn't pay enough attention :)